com.unity.render-pipelines.high-definition/Runtime/Material/BuiltinGIUtilities.hlsl

```diff
@@ -108,7 +108,7 @@ void EvaluateLightmap(float3 positionRWS, float3 normalWS, float3 backNormalWS,
         TEXTURE2D_ARGS(unity_DynamicDirectionality, samplerunity_DynamicLightmap),
         uvDynamicLightmap, unity_DynamicLightmapST, normalWS, backNormalWS, false, decodeInstructions, bakeDiffuseLighting, backBakeDiffuseLighting);
 #else
-    float3 illuminance += SampleSingleLightmap(TEXTURE2D_ARGS(unity_DynamicLightmap, samplerunity_DynamicLightmap), uvDynamicLightmap, unity_DynamicLightmapST, false, decodeInstructions);
+    float3 illuminance = SampleSingleLightmap(TEXTURE2D_ARGS(unity_DynamicLightmap, samplerunity_DynamicLightmap), uvDynamicLightmap, unity_DynamicLightmapST, false, decodeInstructions);
```

Comment:
I will do a separate PR for this fix as it need to be backported, thanks for finding it